README.md

```diff
@@ -7,6 +7,11 @@ The current version is <a href="https://play.google.com/store/apps/details?id=ed
 
 **Please note that the app has not yet been clinically validated.** 
 
+## Publishing
```

Comment:
Unsure if this is the right place to document this?

Reply:
Yep. I think this is a good place for the documentation.